### PR TITLE
Add error message if Xcode 12 is used

### LINKED ISSUE #800: Link for iOS fails with Xcode 12
```
ld: building for iOS, but linking in object file built for macOS
```
for llvm.o

Related to https://github.com/oracle/graal/issues/2857,

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #800

### CODE REVIEW

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=6255043&bid=20368944)

**Review by johanvos — APPROVED** [review]

File: src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java [hunk 1]
```diff
@@ -480,6 +481,21 @@ public void canRunLLVM(Triplet triplet) throws IOException, InterruptedException
...
         }
+
+        // TODO: Remove when GraalVM has support for Xcode 12
+        if (new Triplet(Constants.Profile.IOS).equals(triplet)) {
+            Path gluonversion = graalPath.resolve("gluonversion.txt");
+            if (!Files.exists(gluonversion) ||
+                    !Files.readAllLines(gluonversion).contains("graalvm-ce-java11-20.2.0.hotfix-xcode12")) {
+                XcodeUtils xcodeUtils = new XcodeUtils(XcodeUtils.SDKS.IPHONEOS);
+                if (xcodeUtils.getDTXcode().startsWith("12")) {
+                    throw new IOException("\n\n\nYour current GraalVM build can't build for iOS with Xcode 12+.\n\n" +
```
Comment (tiainen) [inline src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java:492]: `11+` also includes version 12. I would remove the `+` suffixes in the message.

Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java:492]: Where does `12+` relate to 11? I'll remove it anyway

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=6255043&bid=20368944)

File: src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java (hunk 1 above)
Comment (tiainen) [reply src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java]: I was in fact referring to the 11+ in the next line. So having 12+ indeed makes a bit of sense, but let us hope we can remove this code before xcode 13.

Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java]: Oh, my bad, fixed it.

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=6255043&bid=20368944)

**Review by tiainen — APPROVED** [review]

**Pull request merged**